docs/resources/elb_loadbalancer.md

```diff
@@ -0,0 +1,150 @@
+    "cn-north-4b",
+  ]
+  enterprise_project_id = "{{ eps_id }}"
+
+  ipv4_eip_id            = "{{ eip_id }}"
+}
+```
+
+### Loadbalancer With EIP
+
+```hcl
+resource "huaweicloud_elb_loadbalancer" "basic" {
+  name                  = "basic"
+  description           = "basic example"
+  cross_vpc_backend     = true
+
+  vpc_id          = "{{ vpc_id }}"
+  ipv6_network_id = "{{ ipv6_network_id }}"
+  ipv6_bandwidth_id = "{{ ipv6_bandwidth_id }}"
+  ipv4_subnet_id  = "{{ subnet_id }}"
+
+  l4_flavor_id = "{{ l4_flavor_id }}"
+  l7_flavor_id = "{{ l7_flavor_id }}"
+
+  availability_zone = [
+    "cn-north-4a",
+    "cn-north-4b",
+  ]
+  enterprise_project_id = "{{ eps_id }}"
+
+  iptype                = "5_bgp"
+  bandwidth_charge_mode = "traffic"
+  sharetype             = "PER"
+  bandwidth_size        = 10
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the loadbalancer resource.
+    If omitted, the provider-level region will be used.
+    Changing this creates a new loadbalancer.
+
+* `availability_zone` - (Required, List, ForceNew) Specifies the list of AZ names.
+  Changing this parameter will create a new resource.
+
+* `name` - (Optional, String) Human-readable name for the loadbalancer.
+
+* `description` - (Optional, String) Human-readable description for the loadbalancer.
+
+* `cross_vpc_backend` - (Optional, Bool) Enable this if you want to associate the IP addresses of backend servers with your load balancer. Can only be true when updating.
+
+* `vpc_id` - (Optional, String, ForceNew) The vpc on which to create the loadbalancer. Changing this creates a new loadbalancer.
+
+* `ipv4_subnet_id` - (Optional, String) The subnet on which to allocate the loadbalancer's ipv4 address.
+
+* `ipv6_network_id` - (Optional, String) The network on which to allocate the loadbalancer's ipv6 address.
+
+* `ipv6_bandwidth_id` - (Optional, String) The ipv6 bandwidth id. Only support shared bandwidth.
+
+* `ipv4_address` - (Optional, String) The ipv4 address of the load balancer.
+
+* `ipv4_eip_id` - (Optional, String, ForceNew) The ID of the EIP. Changing this parameter will create a new resource.
+
+-> **Note:** If the ipv4_eip_id parameter is configured, you do not need to configure the bandwidth parameters:
+  `iptype`, `bandwidth_charge_mode`, `bandwidth_size` and `share_type`.
+
+* `iptype` - (Optional, String, ForceNew) Elastic IP type. Changing this parameter will create a new resource.
+
+* `bandwidth_charge_mode` - (Optional, String, ForceNew) Bandwidth billing type. Changing this parameter will create a new resource.
+
+* `sharetype` - (Optional, String, ForceNew) Bandwidth sharing type. Changing this parameter will create a new resource.
+
+* `bandwidth_size` - (Optional, Int, ForceNew) Bandwidth size. Changing this parameter will create a new resource.
+
+* `l4_flavor_id` - (Optional, String) The L4 flavor id of the load balancer.
```

Comment:
how users get the value of flavor id? can we provide a data source?

Reply:
will provide flavor data source in the following up PR